ChatClient/src/main/java/org/example/chatclient/ServerManager.java

```diff
@@ -112,4 +119,30 @@ else if (main.getTextColor().getTEXT() == main.getTextColor().getReset() || main
                 System.out.println(main.getTextColor().getBACKGROUND() + main.getTextColor().getTEXT() + message + main.getTextColor().getReset());
         }
     }
+
+    private void playNotificationSound(String notificationType) {
+    String soundFilePath;
+    if (notificationType.equals("message")) {
+        soundFilePath = "src/main/java/org/example/sounds/message_notification.wav";
```

Comment:
Flytta in ljudfilder till src/main/resources/
sedan är soundFilePath från därifrån
så om du lägger ljudfilen i src/main/resources/sounds/notif.wav
så är filePath = "/sounds/notif.wav"